src/Logic/Taxonomy.php

```diff
@@ -214,6 +226,148 @@ public function get_or_create_category_by_name_and_parent_id( string $cat_name,
 		return $cat_id;
 	}
 
+	/**
+	 * Gets or creates a category by its name and parent term_id.
+	 *
+	 * The array is the same as wp_insert_category accepts. https://developer.wordpress.org/reference/functions/wp_insert_category
```

Comment:
A tiny nitpick ;) could we please put this description directly in the @param description, i.e.
```
* @param array  $data              The data to create the category with. The array is the same as wp_insert_category 
*                                  accepts. https://developer.wordpress.org/reference/functions/wp_insert_category.
```

And same thing for the other method `get_or_create_tag` 🙏